scd4j/src/main/java/com/datamaio/scd4j/conf/Configuration.java

```diff
@@ -156,21 +160,14 @@ private synchronized String buildCurrentExecutionPath() {
 			SimpleDateFormat df = new SimpleDateFormat("yyyyMMddHHmmss");
 			
 			Path workspace = workspace();
-			Path config = getConfig();
-			String fileName;
-			if(config!=null) {
-				fileName = workspace.relativize(config).toString().replace("config/", "");
-			} else {
-				fileName = workspace.toString().replace("config/", "");
-			}
-			
+			String fileName = this.getConfig().getFileName().toString();
```

Comment:
For me, a configuration file must be setted. Maybe a exception should be thrown if getConfig had a null value...